game/assets/jak2/game_subtitle.gp

```diff
@@ -6,10 +6,40 @@
     :text-version "jak2"
     :lines "game/assets/jak2/subtitle/subtitle_lines_en-US.json"
     :meta "game/assets/jak2/subtitle/subtitle_meta_en-US.json")
+  (file-json
+    :language-id 1
+    :text-version "jak2"
+    :lines "game/assets/jak2/subtitle/subtitle_lines_en-US.json"
+    :meta "game/assets/jak2/subtitle/subtitle_meta_en-US.json")
```

Comment:
I think this should be done the same way as jak 1.  Use `lines-base` and `meta-base` so that missing translations use the merged-english lines, and when Crowdin is setup, it will create the translatable files.

Reply:
should be good now